docs/architecture-decisions/adr012-catalog-importer-editable-kind-and-type.md

```diff
@@ -0,0 +1,45 @@
+---
+id: adrs-adr012
+title: ADR012: Catalog importer editable kind and type
+description: Architecture Decision Record (ADR) log on which fields are to be made chosable by the catalog importer on repositories
+---
+
+## Background
```

Comment:
I'm not sure if this is worth an ADR.

Reply:
I've hesitated as I've indeed seen very few ADRs in the repository. However, a new contributor might read https://backstage.io/docs/architecture-decisions/adrs-overview which then refers to https://engineering.atspotify.com/2020/04/14/when-should-i-write-an-architecture-decision-record/

The blog post suggests I should indeed make an ADR for this change. So I think we either need to do more ADRs, or to update the documentation if we want to produce fewer. What do you think?

Reply:
Although I love the efforts, this is more of a feature that does not necessarily have an broader technology or architecture impact on the code base, so therefore probably doesn't require or warrant and ADR.

Reply:
Alright, @andrewthauer do you feel like we should update the Docs to make it clear which features require an ADR, and which ones are covered enough through the elements you mention?

Reply:
I would expect an ADR to be created when there is difference of opinion on a matter, debate happens, and then the decision is recorded. In this case it seems like everyone generally agrees and thus the value of adding an ADR is limited.

With that said, personally I don't see a significant cost to adding an ADR so I wouldn't request changes based on this. Others may disagree.

Reply:
Would also say that ADRs are generally how to write code rather than how to implement features. For example what languages to use, code style, libraries, particular patterns to use or avoid, etc. So +1 to not needing an ADR for this change